app/views/csvs/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>CSV Report status</h1>
+
+<section class="clearfix">
+  <div>
+    <% if @csv_export.status? :deleted? %>
+      <p>Your report has been deleted.  <%= link_to "Request a new report.", new_csv_path %></p>
+    <% elsif @csv_export.status? :ready %>
+      <p><%= link_to "Your report is ready for download.", csv_path(@csv_export, format: 'csv') %></p>
+    <% elsif @csv_export.status? :failed %>
+      <p>Your report has failed.  <%= link_to "Request a new report.", new_csv_path %></p>
```

Comment:
can we store the error ... and dump that here ... or say 'please contact and administrator to check the logs'